### DISCUSSION TITLE
Unable to set color to Popover trigger button icon

### DISCUSSION THREAD

**Discussion by spidertwin2** [chakra-ui/chakra-ui#3562] [Q&A] [answered]
hello,

I am unable to pass `color=gray.300` to the PopoverButton component.  

[Link](https://codesandbox.io/s/vibrant-shamir-nmf1o?file=/src/PopoverButton.js) to sandbox.

Also, why is the error `Element type is invalid` being thrown?

Thank you   :dolphin:

**Comment by malashevskyi** [comment, marked as answer]
1. First error. You need to replace ```import { PopoverButton } from "./PopoverButton";``` with ```import PopoverButton from "./PopoverButton";``` because you use default export.
2. Try to set ```color=gray.300``` on TooltipIconButton, not on React Icon (if I understood you correctly)
3. Second error related to @chakra-ui/react, I don't know why but if you change ```1.3.4``` version with ```1.3.3``` then it will work

[working example](https://codesandbox.io/s/chakra-ui-tooltip-popover-1pgjd)

**Reply by spidertwin2** [reply]
ah! makes sense..works now 

Спасибо :whale:
